### ISSUE TITLE
Conditional defines cause syntax highlighting errors

### ISSUE THREAD

**Issue by harm-smits** [dude719/JetBrains-NASM-Language#23]
The following piece of code that I use for prefixing lables with an `_` on macos gives a syntax error on the define after the `%elifidn` directive.

```asm
%ifidn __OUTPUT_FORMAT__, elf64
	%define LABEL_LHS(prefix)
%elifidn __OUTPUT_FORMAT__, macho64
	%define LABEL_LHS(prefix) _
%endif
```

When I hover over the red scribble, I see the following error:

```
<expr>, NASMTokenType.!=, NASMTokenType.(, NASMTokenType.<, NASMTokenType.<=, NASMTokenType.==, NASMTokenType.>, NASMTokenType.>=, NASMTokenType.AVX2_OP, NASMTokenType.AVX512_OP, NASMTokenType.AVX_OP, NASMTokenType.FPU_OP, NASMTokenType.GENERAL_OP, NASMTokenType.MMX_OP, NASMTokenType.SSE2_OP, NASMTokenType.SSE3_OP, NASMTokenType.SSE4_OP, NASMTokenType.SSE_OP, NASMTokenType.SYSTEM_OP, NASMTokenType.VIRTUALIZATION_OP or NASMTokenType.X64_OP expected, got '%define'
```

**Comment by ajkhoury** [comment]
I'm aware of this bug, the language bnf/lex needs to have the `,` token added to the conditional preprocessor definition

**Comment by ajkhoury** [comment]
Fixed in 322f7a7
